src/Constants.java

```diff
@@ -0,0 +1,14 @@
+public class Constants {
+    public static final int UserArgumentNumber = 2;
```

Comment:
please follow the Constants class variable name like this  UserArgumentNumber variable to change USER_ARGUMENT_NUMBER
update all Constants class variable

Reply:
Updated all the Constants class variables.